retrieve_files.sh

```diff
@@ -104,9 +104,6 @@ case "$1" in
   log "ERR" "argument passed in is not valid: $1"
   exit 1
 esac
-     
-#create directory for downloaded files(if does not exist already)
-mkdir -p $dropLocation
```

Comment:
I would leave this line in, even without daily directories... can't hurt, and could avoid a false start on the script in a new environment.